From b8a0e563fd181205565a0edaaebc82b1abf0c5be Mon Sep 17 00:00:00 2001
From: Tim Gover <tim.gover@raspberrypi.com>
Date: Fri, 1 Nov 2024 19:43:21 +0000
Subject: [PATCH] serial: core: Add the Raspberry Pi firmware UART id

Assign a new serial core number for the RPi firmware UART.

Signed-off-by: Tim Gover <tim.gover@raspberrypi.com>
---
 include/uapi/linux/serial_core.h | 3 +++
 1 file changed, 3 insertions(+)

--- a/include/uapi/linux/serial_core.h
+++ b/include/uapi/linux/serial_core.h
@@ -245,4 +245,7 @@
 /* Sunplus UART */
 #define PORT_SUNPLUS	123
 
+/* RPi firmware UART */
+#define PORT_RPI_FW	124
+
 #endif /* _UAPILINUX_SERIAL_CORE_H */
